usb/backend/libusb1.py

```diff
@@ -731,6 +731,13 @@ def get_device_descriptor(self, dev):
         except AttributeError:
             dev_desc.port_numbers = None
 
+        try:
+            dev_desc.speed = self.lib.libusb_get_device_speed(dev.devid)
+            dev_desc.iSpeed = None
```

Comment:
see in http://git.libusb.org/?p=libusb.git;a=blob;f=libusb/libusb.h;h=044012bcd00dbfc5f47d97745681e9e87ef775a7;hb=HEAD;js=1 
714 enum libusb_speed {
 715     /** The OS doesn't report or know the device speed. */
 716     LIBUSB_SPEED_UNKNOWN = 0,
 717 
 718     /** The device is operating at low speed (1.5MBit/s). */
 719     LIBUSB_SPEED_LOW = 1,
 720 
 721     /** The device is operating at full speed (12MBit/s). */
 722     LIBUSB_SPEED_FULL = 2,
 723 
 724     /** The device is operating at high speed (480MBit/s). */
 725     LIBUSB_SPEED_HIGH = 3,
 726 
 727     /** The device is operating at super speed (5000MBit/s). */
 728     LIBUSB_SPEED_SUPER = 4,
 729 };